index.js

```diff
@@ -525,7 +545,7 @@ function makeBee (key, corestore, onwait) {
     : { name: 'db', cache: true, onwait }
   const core = corestore.get(metadataOpts)
   const metadata = { contentFeed: null }
-  return new Hyperbee(core, { keyEncoding: 'utf-8', valueEncoding: 'json', metadata })
+  return new Hyperbee(core, { keyEncoding: DB_KEY_ENCODING, valueEncoding: 'json', metadata })
```

Comment:
would just inline the db_key_encoding personally

Reply:
Went with the constant because we also need it when defining `FILES_SUB`, but fine with explicitly writing `utf-8` for both too